README.md

```diff
@@ -1283,167 +1282,6 @@ Image references should not be updated individually because these images have
-           <a href="https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list">
-           manifest list</a>, the desired operating system version. For example,
-           <code>10.0.10586</code>.</p>
-      </td>
-    </tr>
-    <tr>
-      <td><code>os_features</code></td>
-      <td>
-        <p><code>string list; optional</code></p>
-        <p>When the specified image refers to a multi-platform
-           <a href="https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list">
-           manifest list</a>, the desired operating system features. For example,
-           on Windows this might be <code>["win32k"]</code>.</p>
-      </td>
-    </tr>
-    <tr>
-      <td><code>architecture</code></td>
-      <td>
-        <p><code>string; optional</code></p>
-        <p>When the specified image refers to a multi-platform
-           <a href="https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list">
-           manifest list</a>, the desired CPU architecture. For example,
-           <code>amd64</code> or <code>arm</code>.</p>
-      </td>
-    </tr>
-    <tr>
-      <td><code>cpu_variant</code></td>
-      <td>
-        <p><code>string; optional</code></p>
-        <p>When the specified image refers to a multi-platform
-           <a href="https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list">
-           manifest list</a>, the desired CPU variant. For example, for ARM you
-           may need to use <code>v6</code> or <code>v7</code>.</p>
-      </td>
-    </tr>
-    <tr>
-      <td><code>platform_features</code></td>
-      <td>
-        <p><code>string list; optional</code></p>
-        <p>When the specified image refers to a multi-platform
-           <a href="https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list">
-           manifest list</a>, the desired features. For example, this may
-           include CPU features such as <code>["sse4", "aes"]</code>.</p>
-      </td>
-    </tr>
-    <tr>
-      <td><code>docker_client_config</code></td>
-      <td>
-        <p><code>string; optional</code></p>
-        <p>Specifies the directory to look for the docker client configuration. Don't use this directly.
-           Specify the docker configuration directory using a custom docker toolchain configuration. Look
-           for the <code>client_config</code> attribute in <code>docker_toolchain_configure</code> <a href="#setup">here</a> for
-           details. See <a href="#container_pull-custom-client-configuration">here</a> for an example on
-           how to use <code>container_pull</code> after configuring the docker toolchain</p>
-        <p>When left unspecified (ie not set explicitly or set by the docker toolchain), docker will use
-        the directory specified via the DOCKER_CONFIG environment variable. If DOCKER_CONFIG isn't set,
-        docker falls back to $HOME/.docker.
-        </p>
-      </td>
-    </tr>
-  </tbody>
-</table>
-
```

Comment:
add puller_darwin and puller_linux docs somewhere down here

Reply:
done